03-csv-vector-raster-plotting.Rmd

```diff
@@ -284,18 +278,29 @@ plot(plot.locationsSp_HARV,
 
 ##Define Plot Extent
 
-In lesson XXX <link here> we learned about spatial object `extent`. When we plot
-several spatial layers in `R`, the first layer that is plotted, becomes the extent
-of the plot. If we add additional layers that are outside of that extent, then
-the data will not be visible in our plot. It is thus useful to know how to set the 
-spatial extent of a plot using `xlims` and `ylims.
+In 
+[Lesson 00: Open and Plot Shapefiles in R]({{site.baseurl}}/R/open-shapefiles-in-R/)
+we learned about spatial object `extent`. When we plot several spatial layers in
+`R`, the first layer that is plotted, becomes the extent of the plot. If we add
+additional layers that are outside of that extent, then the data will not be
+visible in our plot. It is thus useful to know how to set the spatial extent of
+a plot using `xlims` and `ylims`.
+
+Let's create a boundary for our area of interest (AOI) for the Harvard field
+site.  (If you have completed Lesson 00 or 01 in this series you can skip this
+chunk as you have already created this object.)
```

Comment:
You can skip this "code". -- remember that they wont' see the chunks like we see them. it is just code on a page.